src/components/breeds/Breeds.jsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useEffect, useState } from "react";
+import { api } from "../../utils/api";
+import BreedItem from "./BreedItem";
+import { SearchBar } from "../searchBar/SearchBar";
+import ReactPaginate from "react-paginate";
+import { motion } from "framer-motion";
+
```

Comment:
Yes the reason is forgot to drop it fragments by mistake the This will be fixed if i do this
 return (
      <Link to={breedLink}>
        <h3> {breed}</h3>
        {imageBreed && <img src={imageBreed} />}
      </Link>
  );
};